retrofit-converters/wire/src/main/java/retrofit/WireResponseBodyConverter.java

```diff
@@ -30,11 +30,16 @@
   }
 
   @Override public T convert(ResponseBody value) throws IOException {
+    BufferedSource source = value.source();
```

Comment:
calling `value.source()` can throw, so this will need split into two lines, variable initialization and retrieving the source–the latter of which is inside the `try` block